### DISCUSSION TITLE
monotype in parsed-literal directive rendered with a smaller font

### DISCUSSION THREAD

**Discussion by dvarrazzo** [pradyunsg/furo#273] [Potential Issue] [answered]
The following two blocks should be rendered in a similar fashion:
```
    .. code-block::

        asyncio.set_event_loop_policy(
            asyncio.WindowsSelectorEventLoopPolicy()
        )

    .. parsed-literal::

        `asyncio.set_event_loop_policy`\ (
            `asyncio.WindowsSelectorEventLoopPolicy`\ ()
        )

        also test *italic*, **bold**, ``monotype``.
```

Instead, monotype appears smaller than the rest of the text:

![image](https://user-images.githubusercontent.com/199429/135161188-ddc3fbe4-18c0-4696-b294-b6d90d3acb61.png)

**Comment by pradyunsg** [comment]
<img width="783" alt="Screenshot 2021-09-28 at 22 22 09" src="https://user-images.githubusercontent.com/3275593/135167568-c5ede5aa-36e4-4096-9cb5-9909990c89b5.png">


```rst

Test
====

.. code-block::

    asyncio.set_event_loop_policy(
        asyncio.WindowsSelectorEventLoopPolicy()
    )

.. parsed-literal::

    `asyncio.set_event_loop_policy`\ (
        `asyncio.WindowsSelectorEventLoopPolicy`\ ()
    )

    also test *italic*, **bold**, ``monotype``.
```

I'm unable to reproduce this. Could you provide a more complete reproducer? What OS + Browser + zoom combination is this with?

**Reply by dvarrazzo** [reply]
You can see even in your version that the "monotype" word is smaller.

My example is generated with `default_role = "obj"` in `conf.py`. You can probably reproduce the same with
```
:obj:`asyncio.set_event_loop_policy`
```
or anything to the same effect.

A live example is visible [here](https://www.psycopg.org/psycopg3/docs/advanced/async.html).

**Reply by dvarrazzo** [reply]
This is Firefox 91.0.2 at 100% zoom btw.

**Reply by pradyunsg** [reply, marked as answer]
OK, this took me a while to resolve but done with f8db95bed5d1c57b4ea6bf536c65560fa9ae33a9
